imagery/i.ortho.photo/lib/find_init.c

```diff
@@ -5,14 +5,21 @@
  **************************************************************/
 #include <grass/gis.h>
 
+#define ELEMENT_BUFFER_SIZE 80
+
 int I_find_initial(char *group)
 {
     char *element;
+    int file_exists;
 
-    element = (char *)G_malloc(80 * sizeof(char));
-
-    if (group == NULL || *group == 0)
+    if (group == NULL || *group == 0) {
         return 0;
-    sprintf(element, "group/%s", group);
-    return G_find_file(element, "INIT_EXP", G_mapset()) != NULL;
+    }
+
+    element = (char *)G_malloc(ELEMENT_BUFFER_SIZE * sizeof(char));
```

Comment:
The GNAME_MAX discussed in other PR applies here too. No need to introduce another constant. Technically, INAME_LEN should be used for a group name, but I think we will remove INAME_LEN in favor of GNAME_MAX, so here use GNAME_MAX.

If anyone thinks otherwise, please let me know.

Reply:
Done